senticnet/senticnet.py

```diff
@@ -1,21 +1,21 @@
 import rdflib
+import urllib2
 from rdflib.term import URIRef
 
 
 class Senticnet(object):
     """
-    Simple API to use Senticnet 3 without be bothered by RDF stuff.
+    Ultra simple API to use Senticnet 2 without be bothered by RDF stuff.
```

Comment:
I changed it back in the latest commit that is part of this pull request. The final code has the comment with Senticnet 3. I was using the PyPI version initially which had Senticnet 2.